airflow/providers/apprise/hooks/apprise.py

```diff
@@ -21,9 +21,10 @@
 from typing import Any, Iterable
 
 import apprise
-from airflow.hooks.base import BaseHook
 from apprise import AppriseConfig, NotifyFormat, NotifyType
```

Comment:
As an example: these `apprise` imports were incorrectly marked as first-party, since `apprise` was missing from the `known-third-party`.